.gitignore

```diff
@@ -36,7 +37,6 @@
 build_*.err
 build_*.log
 build_*.wrn
-Build/.vs/
```

Comment:
Exactly. Lines in .gitignore apply to any subdirectory unless you prefix it with a "root" directory `/`.